Add section about devfile v1 support (#3)
benoitf/asciidoc-guides: start-workspace-from-url.adoc

Example: `https://che-server#https://github.com/example/foo?policies.create=peruser` to create or reopen the workspace created from this repository.

== Support of devfiles

> {prod-short} uses the {devworkspace} engine that targets v2 devfiles. (SchemaVersion is 2.1.0)

Reviewer: ```suggestion
{prod-short} uses the {devworkspace} engine that targets devfiles v2 with `schemaVersion: 2.1.0`.
```

> Using devfile v1 is still possible but deprecated. Under the hood, https://github.com/che-incubator/devfile-converter[converter library], converts devfiles v1 to devfiles v2.

Reviewer: ```suggestion
Use of devfile v1 is deprecated. Under the hood, https://github.com/che-incubator/devfile-converter[converter library] converts devfiles v1 to devfiles v2.
```
